migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,314 @@
+-- Enums
+
+CREATE TYPE gender                  AS ENUM ('male', 'female');
+CREATE TYPE role                    AS ENUM ('user', 'admin');
+CREATE TYPE status                  AS ENUM ('available', 'unavailable');
+CREATE TYPE association             AS ENUM ('sponsor', 'organization', 'media');
+CREATE TYPE task_priority           AS ENUM ('low', 'medium', 'high');
+CREATE TYPE acceptance_status       AS ENUM ('pending', 'accepted', 'rejected');
+CREATE TYPE employee_level          AS ENUM ('basic', 'organizer', 'manager');
+CREATE TYPE employee_contract       AS ENUM ('full-time', 'part-time', 'temporary');
```

Comment:
Consider if this shouldn't reflect the basic types of contracts in the Czech Republic.
In the ERD, this enum has 'DPP', 'DPC' and 'HPP' instead because it may help with the calculation of the timesheet's output.